pkgs/development/node-packages/overrides.nix

```diff
@@ -367,12 +367,11 @@ final: prev: {
 
     src = fetchurl {
       url = "https://registry.npmjs.org/prisma/-/prisma-${version}.tgz";
-      hash = "sha256-0NxYp+W2KbR3xEV2OCXCIL3RqkvLfJHNKgl/PxapVbI=";
+      hash = "sha256-gikKT2iUdSb7/qOpJNiukbm35ggLcI2Y24uuGIqpSbE=";
     };
     postInstall = with pkgs; ''
       wrapProgram "$out/bin/prisma" \
         --set PRISMA_SCHEMA_ENGINE_BINARY ${prisma-engines}/bin/schema-engine \
-        --set PRISMA_MIGRATION_ENGINE_BINARY ${prisma-engines}/bin/schema-engine \
```

Comment:
Deprecated env var, not necessary anymore (it's now covered by `PRISMA_SCHEMA_ENGINE_BINARY` above).